build.sbt

```diff
@@ -74,17 +74,9 @@ def altPublishSettings: Seq[Setting[_]] =
       val moduleSettings = (Keys.moduleSettings).value
       val ivy = new IvySbt((ivyConfiguration.value))
 
-      val module =
-        new ivy.Module(moduleSettings)
-      val newConfig =
-        new PublishConfiguration(
-          config.ivyFile,
-          altLocalRepoName,
-          config.artifacts,
-          config.checksums,
-          config.logging
-        )
-      streams.value.log.info("Publishing " + module + " to local repo: " + altLocalRepoName)
+      val module = new ivy.Module(moduleSettings)
+      val newConfig = config withResolverName altLocalRepoName withOverwrite false
```

Comment:
Nitpick: I know you're a fan of infix, but can we have parenthesis for the first apply here? I find it difficult to read and parse in my mind.